docs/data/material/discover-more/showcase/appList.js

```diff
@@ -27,7 +27,7 @@ const appList = [
     description:
       'QuintoAndar is a company that uses technology and ' +
       'design to simplify the rental of residential real estate.',
-    image: 'quintoandar.png',
+    image: 'quintoandar.jpg',
```

Comment:
It was looking like crap, so redo a few screenshots, following a process close to https://www.notion.so/mui-org/Store-Screenshots-for-product-7a4cf2df5cb94389ab6adc80b6aeb054. We would need to retake all the screenshots, but since we might rework from scratch this page, I only did the first two.